test/functional/change_stream.test.js

```diff
@@ -1552,48 +1636,6 @@ describe('Change Streams', function() {
     }
   });
 
-  it('Should resume after a killCursors command is issued for its child cursor', {
```

Comment:
if this is covered by a spec test can you call that out please?

Reply:
I think that this test is no longer valid, because the previously resumable`CursorNotFound` error is no longer resumable.